### PR TITLE
Bug 1801367: Fix test-e2e make target

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
On line 38 fix file name 0100namespace.yaml -> 0100namespace.yaml

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Currently fixing the dependency graph to fix failing tests

**Comment by zvonkok** [comment]
We are going to cherrypick this in 4.3 so please update the commit to fix only the e2e target. No additional features. Create a separate PR for code coverage and GO_TEST_ARGS.

File: Makefile [hunk 1]
```diff
@@ -34,16 +36,20 @@ all: build
 build:
 	$(GO_BUILD_RECIPE)
 
+test:
+	go test -race -v -cover=1 ./cmd/... ./pkg/... -coverprofile $(COVERPROFILE)
+
 test-e2e: 
-	@${TEMPLATE_CMD} manifests/0110_namespace.yaml > manifests/operator-init.yaml
-	echo -e "\n---\n" >> manifests/operator-init.yaml
+	@${TEMPLATE_CMD} manifests/0100_namespace.yaml > manifests/operator-init.yaml
```
Comment (zvonkok) [inline Makefile]: Only fix the 0100_namespace.yaml in this PR

Comment (ArangoGutierrez) [reply Makefile]: working on it

Comment (ArangoGutierrez) [reply Makefile]: > 64b41f0

**Comment by ArangoGutierrez** [comment]
I have a patch for the `go.mod` clash, if the latest `retest` doesn't work, I'll push it to my branch

**Comment by ArangoGutierrez** [comment]
@zvonkok PR is ready for your review

**Comment by ArangoGutierrez** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1801367

**Pull request merged**